MUXSDKStats/MUXSDKStats/MUXSDKStats.m

```diff
@@ -127,6 +137,53 @@ + (void)initSDK {
 
 #pragma mark Monitor AVPlayerViewController
 
++ (MUXSDKPlayerBinding *_Nullable)monitorAVPlayerViewController:(nonnull AVPlayerViewController *)player
+                                                 withPlayerName:(nonnull NSString *)name
+                                                   customerData:(nonnull MUXSDKCustomerData *)customerData {
+    return [self monitorAVPlayerViewController:player
+                                withPlayerName:name
+                                  customerData:customerData
+                        automaticErrorTracking:true];
+}
+
++ (MUXSDKPlayerBinding *_Nullable)monitorAVPlayerViewController:(nonnull AVPlayerViewController *)player
+                                                 withPlayerName:(nonnull NSString *)name
+                                                   customerData:(nonnull MUXSDKCustomerData *)customerData
+                                         automaticErrorTracking:(BOOL)automaticErrorTracking {
+
+    [self initSDK:[customerData customerViewerData]];
```

Comment:
note that customerViewer data is included and overwritten with _every_ monitorAVPlayer* call